### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file, new_file = command.split(" ")[1:]
+
+    if old_file != new_file:
```
Comment (mltdll) [inline app/main.py:4]: Putting the function's main logic inside an `if` condition is usually considered a bad practice.

Also, the function shouldn't copy file if the command isn't `cp`.

**Review by mltdll — CHANGES_REQUESTED** [review]
You sho

File: app/main.py [hunk 2]
```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(line: str) -> str:
+    command, old_file, new_file = line.split(" ")[1:]
+
+    if command != "cp" or old_file == new_file:
+        return "Incorrect command or your file names are the same!"
```
Comment (mltdll) [inline app/main.py:5]: I'd say that the function shouldn't return this string. You should either use empty `return` statement or raise an error with this message.

**Review by mltdll — CHANGES_REQUESTED** [review]

**Review by mltdll — APPROVED** [review]

File: app/main.py (hunk 2 above)
Comment (HStakhiv) [reply app/main.py]: Thank you!

**Review by Y-Havryliv — APPROVED** [review]
Nice!

**Pull request closed without merging**